app/controllers/curation_concerns/curation_concerns_controller.rb

```diff
@@ -2,12 +2,23 @@ class CurationConcerns::CurationConcernsController < ApplicationController
   include CurationConcerns::CurationConcernController
 
   def create
-    super
-  rescue StandardError => err
-    metadata_id = params[curation_concern.class.name.underscore]['source_metadata_identifier']
-    logger.debug "Error retrieving metadata for #{metadata_id}: #{err}"
-    flash[:alert] = "Error retrieving metadata for '#{metadata_id}'"
-    render :new, status: 500
+    begin
+      super
+    rescue StandardError => err
+      metadata_id = params[curation_concern_name]['source_metadata_identifier']
+      logger.debug "Error retrieving metadata for #{metadata_id}: #{err}"
+      flash[:alert] = "Error retrieving metadata for '#{metadata_id}'"
+      render :new, status: 500
+    end
+
+    return unless params[curation_concern_name][:parent_id]
+    parent = ActiveFedora::Base.find(params[curation_concern_name][:parent_id], cast: true)
+    parent.members << curation_concern.reload
```

Comment:
Should we wrap these interactions up in an "Actor", as CC does? I'm a bit ambivalent, as I like seeing the logic in the controller action, but there may be strengths to following a similar pattern. Thoughts?